authmanager/authmanager.go

```diff
@@ -24,6 +24,11 @@ const (
 
 // AcquireToken stores the issue tracker's auth token based on the auth type specified
 func AcquireToken(cfg *config.Local) error {
+
+	if !cfg.Auth.Type.IsValid() {
+		return fmt.Errorf("invalid auth type: %q. valid auth types are: %q", cfg.Auth.Type, config.ValidAuthTypes)
+	}
```

Comment:
Could you also remove this validation from `validation.Validate`?

There's no value in having in in two places.